docs/general/events.md

```diff
@@ -17,74 +17,55 @@ network connection severed, etc.
 
 In OpenTelemetry, Events are implemented as a specific type of [`LogRecord`](https://github.com/open-telemetry/opentelemetry-specification/tree/v1.44.0/specification/logs/api.md) that conforms to the conventions included here.
 
-In addition to a required name, an Event may contain a _payload_ (body) of any type permitted
-by the [LogRecord body](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/logs/data-model.md#field-body).
-In its implementation, the Event _payload_ (body) will constitute the `Body` of the `LogRecord`.
-Like all other OpenTelemetry signals, an Event has optional attribute metadata that helps describe
-the event context.
+Events SHOULD be generated / produced / recorded using the
+[Logs API](https://github.com/open-telemetry/opentelemetry-specification/tree/v1.44.0/specification/logs/api.md#emit-a-logrecord)
+to ensure that the event is created using the configured SDK instance.
```

Comment:
nit: The user does not need to use OTel SDK (it is fine to use a custom Logs API implementation). I suggest just removing this sentence to avoid confusion.

```suggestion
[Logs API](https://github.com/open-telemetry/opentelemetry-specification/tree/v1.44.0/specification/logs/api.md#emit-a-logrecord).
```

Reply:
I removed the sentence, we never prescribe how to create telemetry. you can create valid OTLP LogRecord in the collector as far as semconvs is concerned